src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureArea, Colorable {
```

Comment:
Don't use abstract classes to set behavior for classes. In this case, you've used an abstract class to implement interfaces for behavior. Consider whether you need an abstract class or you can simply implement the interfaces in your concrete classes.